site/_includes/partials/chrome-100-card-section.njk

```diff
@@ -0,0 +1,2469 @@
+  <div class="navigation-wrapper">
+    <img src="/images/meta/chrome_2008.png" alt="Chrome logo" class="chrome-logo">
+    <nav class="year-navigation">
+      <ul class="menu">
+        <li id="menu__section2008" class="active menuitem">
+          <a class="sectionLink" data-index="0" href="#section2008">2008</a>
+        </li>
+        <li id="menu__section2009" class="menuitem">
+          <a class="sectionLink" data-index="1" href="#section2009">2009</a>
+        </li>
+        <li id="menu__section2010" class="menuitem">
+          <a class="sectionLink" data-index="2" href="#section2010">2010</a>
+        </li>
+        <li id="menu__section2011" class="menuitem">
+          <a class="sectionLink" data-index="3" href="#section2011">2011</a>
+        </li>
+        <li id="menu__section2012" class="menuitem">
+          <a class="sectionLink" data-index="4" href="#section2012">2012</a>
+        </li>
+        <li id="menu__section2013" class="menuitem">
+          <a class="sectionLink" data-index="5" href="#section2013">2013</a>
+        </li>
+        <li id="menu__section2014" class="menuitem">
+          <a class="sectionLink" data-index="6" href="#section2014">2014</a>
+        </li>
+        <li id="menu__section2015" class="menuitem">
+          <a class="sectionLink" data-index="7" href="#section2015">2015</a>
+        </li>
+        <li id="menu__section2016" class="menuitem">
+          <a class="sectionLink" data-index="8" href="#section2016">2016</a>
+        </li>
+        <li id="menu__section2017" class="menuitem">
+          <a class="sectionLink" data-index="9" href="#section2017">2017</a>
+        </li>
+        <li id="menu__section2018" class="menuitem">
+          <a class="sectionLink" data-index="10" href="#section2018">2018</a>
+        </li>
+        <li id="menu__section2019" class="menuitem">
+          <a class="sectionLink" data-index="11" href="#section2019">2019</a>
+        </li>
+        <li id="menu__section2020" class="menuitem">
+          <a class="sectionLink" data-index="12" href="#section2020">2020</a>
+        </li>
+        <li id="menu__section2021" class="menuitem">
+          <a class="sectionLink" data-index="13" href="#section2021">2021</a>
+        </li>
+        <li id="menu__section2022" class="menuitem">
+          <a class="sectionLink" data-index="14" href="#section2022">2022</a>
+        </li>
+      </ul>
+    </nav>
+  </div>
+
+  <div class="side-arrow-DT">
+    <img src="/images/project/arrow3.svg" alt="Up arrow" class="side-arrow">
+    <img src="/images/project/arrow4.svg" alt="Down aroow" class="side-arrow">
+  </div>
+
+  {# Year 2008 #}
+  <section class="inner-card display-flex center-margin gap-right-300 year-2008" id="section2008" data-index="0">
+    <div class="first-column gap-right-300">
+      <div class="card card-with-img gap-bottom-300">
+        <img src="/images/project/card-img.svg" alt="Placeholder" class="card-img">
+        <div class="card-info">
```

Comment:
Given how repetitive this structure is, refactoring into a helper partial might have made more sense. It's not required, but it could save you some time if you ever need to, e.g., tweak the layout by making change to the partial once, instead of many changes to each element on this page.

Reply:
Yes, we are agree modifying it at this point could be little risky and will require more efforts so keeping it as is. If needed it could be altered post launch.